docs/guide/browser/component-testing.md

```diff
@@ -261,23 +261,20 @@ test('ShoppingCart manages items correctly', async () => {
 ```tsx
 // Option 1: Recommended - Use MSW (Mock Service Worker) for API mocking
 import { http, HttpResponse } from 'msw'
-import { setupServer } from 'msw/node'
+import { setupWorker } from 'msw/browser'
 
-// Set up MSW server with API handlers
-const server = setupServer(
+// Set up MSW worker with API handlers
+const worker = setupWorker(
   http.get('/api/users/:id', ({ params }) => {
-    const { id } = params
-    if (id === '123') {
-      return HttpResponse.json({ name: 'John Doe', email: 'john@example.com' })
-    }
-    return HttpResponse.json({ error: 'User not found' }, { status: 404 })
+    // Describe the happy path
+    return HttpResponse.json({ id: params.id, name: 'John Doe', email: 'john@example.com' })
```

Comment:
Also recommending simpler happy path setups. Developers can always make it conditional but the example shouldn't care about that.